pkg/kubectl/resource/builder.go

```diff
@@ -274,12 +275,48 @@ func (b *Builder) SelectAllParam(selectAll bool) *Builder {
 	return b
 }
 
+func hasNamesArg(args []string) bool {
+	if len(args) > 1 {
+		return true
+	}
+	newArgs := []string{}
+	for _, s := range args {
+		newArgs = append(newArgs, strings.Split(s, ",")...)
```

Comment:
Because there might be resource1,group2/resource2 name. We need to separate by `,` first and then by `/`.